storage/google/cloud/storage/client.py

```diff
@@ -194,7 +194,7 @@ def lookup_bucket(self, bucket_name):
         except NotFound:
             return None
 
-    def create_bucket(self, bucket_name):
+    def create_bucket(self, bucket_name, requester_pays=None):
```

Comment:
The feature is especially convenient for this API, which actually creates the bucket on the server side.  Unlike the many other properties, `requester_pays` has an impact on *every* subsequent use of the bucket after the initial `create` call.  Apps which construct buckets via `Client.bucket` or `Bucket.__init__` can set the property before calling `bucket.create`.